haas/ext/switches/brocade.py

```diff
@@ -60,33 +60,41 @@ def session(self):
     def disconnect(self):
         pass
 
-    def apply_networking(self, action):
+    def modify_port(self, port, channel, network_id):
         """ Apply a NetworkingAction to the switch.
```

Comment:
@zenhack can you update the comment a bit here? I don't mind getting rid of it or else.